src/main/kotlin/no/nav/familie/ba/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -511,6 +508,15 @@ class FagsakService(
                 )
             }
 
+    private fun settEgenAnsattStatusPåFagsakDeltagere(fagsakDeltagere: MutableList<RestFagsakDeltager>): List<RestFagsakDeltager> {
```

Comment:
Denne trenger vel kanskje ikke å være `mutable`? 

```suggestion
    private fun settEgenAnsattStatusPåFagsakDeltagere(fagsakDeltagere: List<RestFagsakDeltager>): List<RestFagsakDeltager> {
```